Research/Research/RSDImageThemeObject.swift

```diff
@@ -63,7 +63,7 @@ public struct RSDFetchableImageThemeElementObject : RSDFetchableImageThemeElemen
     }
 
     /// The name of the image.
-    public let imageName: String
+    public let imageName: RSDFileWrapper
```

Comment:
Should this property name be changed to prevent confusion? Or would there be tricky ripple effects from doing that?

Reply:
Tricky ripply effect. RSDFileWrapper implements the StringLiteral protocol (it's effectively a String) so by keeping the name the same, it will decode the String to a file wrapper. This allows for reverse-compatibility without having to include deprecation warnings.